File: game-admin/src/main/java/com/dkm/feign/entity/UserLoginStatusVo.java
package com.dkm.feign.entity;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.jetbrains.annotations.Nullable;


@NoArgsConstructor
@AllArgsConstructor
@Data
public class UserLoginStatusVo {
    @Nullable
    private String cid;
    @Nullable
    private ResultUser resultUser;
    @Nullable
    private String token;


}

File: game-admin/src/main/java/com/dkm/feign/entity/UserLoginStatusVo.kt
package com.dkm.feign.entity

data class UserLoginStatusVo(
    val cid: String,
    val resultUser: ResultUser,
    val token: String
)

data class ResultUser(
    val userGold: Int,
    val userGrade: Int,
    val userId: String,
    val userIdCard: Any,
    val userIsEffective: Int,
    val userIsVip: Int,
    val userPacketBalance: Int,
    val userPhone: Any,
    val userRemark: String,
    val userRenown: Int,
    val weChatHeadImgUrl: Any,
    val weChatNickName: String,
    val weChatOpenId: String
)